# Broken App Issues
let axios = require('axios'); -> const axios = require('axios');
var app = express(); -> const app = express();

function -> async function

added "await Promise.all"





catch -> catch (err)